### PR TITLE
Document REACTION_MAX_USER_NUM setting option

### PR DESCRIPTION
**Pull request by 6543**
https://github.com/go-gitea/gitea/blob/4a0103fa293cc42075d4a9c10ca4dffa9c2d3a40/modules/setting/ui.go#L24

https://github.com/go-gitea/gitea/blob/4a0103fa293cc42075d4a9c10ca4dffa9c2d3a40/modules/setting/ui.go#L77

---
*Sponsored by Kithara Software GmbH*

### CODE REVIEW

**Review by techknowlogick — APPROVED** [review]

File: custom/conf/app.example.ini [hunk 1]
```diff
@@ -1221,6 +1221,9 @@ LEVEL = Info
 ;; For custom reactions, add a tightly cropped square image to public/assets/img/emoji/reaction_name.png
 ;REACTIONS = +1, -1, laugh, hooray, confused, heart, rocket, eyes
 ;;
+;; Change the users witch are displayed on mouse hover over (via tooltip) on reactions.
```
Comment (sebastian-sauer) [inline custom/conf/app.example.ini]: I think the information that this is the number of users is missing. What about:

Change the number of users that are displayed in reactions tooltip (triggered by mouse hover)

Comment (6543) [reply custom/conf/app.example.ini]: I'm all  for better wording suggestions

Comment (6543) [inline custom/conf/app.example.ini]:
```suggestion
;; Change the number of users that are displayed in reactions tooltip (triggered by mouse hover).
```

File: docs/content/administration/config-cheat-sheet.en-us.md [hunk 2]
```diff
@@ -223,6 +223,7 @@ The following configuration set `Content-Type: application/vnd.android.package-a
 - `REACTIONS`: All available reactions users can choose on issues/prs and comments
     Values can be emoji alias (:smile:) or a unicode emoji.
     For custom reactions, add a tightly cropped square image to public/assets/img/emoji/reaction_name.png
+- `REACTION_MAX_USER_NUM`: **10**: Change the users witch are displayed on mouse hover over (via tooltip) on reactions.
```
Comment (6543) [inline docs/content/administration/config-cheat-sheet.en-us.md]:
```suggestion
- `REACTION_MAX_USER_NUM`: **10**: Change the number of users that are displayed in reactions tooltip (triggered by mouse hover).
```

**Review by KN4CK3R — APPROVED** [review]

**Comment by silverwind** [comment]
Why don't I find this string `REACTION_MAX_USER_NUM` anywhere in the code base?

**Comment by 6543** [comment]
https://github.com/go-gitea/gitea/blob/4a0103fa293cc42075d4a9c10ca4dffa9c2d3a40/modules/setting/ui.go#L24

https://github.com/go-gitea/gitea/blob/4a0103fa293cc42075d4a9c10ca4dffa9c2d3a40/modules/setting/ui.go#L77

@silverwind

**Comment by KN4CK3R** [comment]
@silverwind You only see the names if the value needs special handling. Otherwise it's just the naming convention for auto-mapping.

**Comment by silverwind** [comment]
Imho dis case convertion should be removed to make these values greppable.

**Comment by KN4CK3R** [comment]
But it's the default. You would not read json into a `map[string]any` and map the fields manually just to make the key names greppable.

**Comment by silverwind** [comment]
Bad default. Nothing prevents us from having `MACRO_CASE`. I do it in other applications as well. Greppability > Variable name conventions :)

**Comment by KN4CK3R** [comment]
Ah, you talk about the field names in the settings structs.

**Pull request merged**